worker/peergrouper/publish.go

```diff
@@ -47,7 +47,7 @@ func (s *CachingAPIHostPortsSetter) SetAPIHostPorts(apiServers [][]network.HostP
 	return nil
 }
 
-func apiServersEqual(a, b [][]network.HostPort) bool {
+func apiServersEqual(a, b [][]network.SpaceHostPort) bool {
```

Comment:
s/[][]network.SpaceHostPort/[]network.SpaceHostPorts/

Reply:
Actually deleted this. It was unused.